Scripts/RemediationScripts/Remediate-EnableAADAuthentication.ps1

```diff
@@ -0,0 +1,708 @@
+<###
```

Comment:
Please rename this file to have SQL somewhere, bcz AAD can be configured for multiple services.